Regression/Checksum/benchmarks_json/test_2d_langmuir_multi.json

```
@@ -3,9 +3,9 @@
     "Bx": 0.0,
     "By": 5.726296856755232,
     "Bz": 0.0,
-    "Ex": 3751589134191.326,
+    "Ex": 3.751589134191326,
     "Ey": 0.0,
-    "Ez": 3751589134191.332,
+    "Ez": 3.751589134191332,
```

Restore correct checksums:
```suggestion
    "Ez": 3751589134191.332,
```